src/index.js

```diff
@@ -1,12 +1,37 @@
 import assert from 'assert';
 import Redis from 'ioredis';
 
+const Ttl = {
+  oneMinute: 60,
+  fiveMinutes: 60 * 5,
+  thirtyMinutes: 60 * 30,
+  oneHour: 60 * 60,
+  fourHours: 60 * 60 * 4,
+  eightHours: 60 * 60 * 8,
+  oneDay: 60 * 60 * 24,
+};
+
+async function getOrSetJSON(redisClient, key, ttlSeconds, valueFunction) {
```

Comment:
This function name seems a little odd. I don't know if it is immediately clear what it is doing. If this is a common practice and I just havnt been exposed to it before then I am good with it.